helix-rest/src/main/java/org/apache/helix/rest/server/resources/metadatastore/MetadataStoreDirectoryAccessor.java

```diff
@@ -152,8 +155,13 @@ public Response getShardingKeys(@QueryParam("realm") String realm) {
   @Path("/metadata-store-realms/{realm}/sharding-keys/{sharding-key: .+}")
   public Response addShardingKey(@PathParam("realm") String realm,
       @PathParam("sharding-key") String shardingKey) {
+    if (shardingKey.charAt(0) != '/') {
+      shardingKey = "/" + shardingKey;
+    }
```

Comment:
Could you add some comments about the Jersey behavior here for reference as a block comment?

Reply:
Fixed together with the later occurrence.